sympy/sets/indicator.py

```diff
@@ -0,0 +1,58 @@
+from sympy.core.symbol import Symbol
+from sympy.core.function import Function
+from sympy.ntheory.residue_ntheory import mobius
+
+from .sets import Set
+
+
+class indicator(Function):
+    r"""
+    Indicator function (also called the Characteristic function) maps an element, x, to the range {0, 1} for set A
+
+    It is defined as follows:
+        1) `1` if x exists in A
+        2) `0` if x does not exist in A
+
+    Parameters
+    ==========
+
+    x : the element
+    a : the set
+
+    Examples
+    ========
+
+    >>> from sympy.sets import Union, Interval, FiniteSet, indicator
+    >>> set = Union(Interval(0, 1), Interval(2, 3))
```

Comment:
Don't use `set` as a variable name. This overwrites the built-in `set`, so it's a bad practice.